R/expr-lang.R

```diff
@@ -203,17 +231,17 @@ is_binary_lang <- function(x, name = NULL, ns = NULL) {
 #' # You can also modify quosures inplace:
 #' f <- ~matrix(bar)
 #' lang_modify(f, quote(foo))
-lang_modify <- function(.call = caller_frame(), ..., .standardise = FALSE) {
+lang_modify <- function(.lang = caller_frame(), ..., .standardise = FALSE) {
```

Comment:
I wonder if these have diverged sufficiently from the original function that the first argument should no longer have a default.